docs/public/installation/prerequisites.md

```diff
@@ -9,14 +9,14 @@ and processor architecture listed below.
 
 ## Supported operating system versions
 
-* [Debian] 12
+* [Debian] 12.8
```

Comment:
We are not consistent. On Rocky Linux we specify the minor version. We should maybe change the wording from "prerequisite" to "this version was release tested on these OS versions" and then minor could be relevant.

Reply:
Yes, I like the idea of saying “we tested Zonemaster on the following OSes”. Better yet, we could state that the version numbers we list on the page are those of the installation media at the date of the last campaign of release testing.

Reply:
Instead of the Prequisites page we could include the versions in the new release page, https://github.com/zonemaster/zonemaster/blob/master/docs/public/RELEASE.md.

Reply:
Just the OS. The difference is subtle and pertains to Debian-based OSes only: when they release a new point version (e.g. 12.8), the [announcement](https://lists.debian.org/debian-announce/2024/msg00008.html) states:

> Please note that the point release does not constitute a new version of Debian 12 but only updates some of the packages included. There is no need to throw away old "bookworm" media. After installation, packages can be upgraded to the current versions using an up-to-date Debian mirror.

So while the installation media is updated so that users installing Debian from offline media won’t lag too far behind current software versions, for existing installations, there is no particular procedure and upgrading with `apt upgrade` will give them the same software as if they had installed the OS from that install media.